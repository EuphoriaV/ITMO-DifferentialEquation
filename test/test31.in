3.0
((((10.3 * 9.85) - (2.32 * x)) * (x + x)) + x)
5.95
1.73