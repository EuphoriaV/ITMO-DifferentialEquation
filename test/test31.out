6.5E-6